encoding/geojson/geojson_test.go

```diff
@@ -287,3 +287,15 @@ func TestFeatureCollection(t *testing.T) {
 		}
 	}
 }
+
+func TestDecodeGeoJSON(t *testing.T) {
+	test := `{"type":"Point","coordinates":[-71.09981854336587,42.327161019017154]}`
```

Comment:
This looks like valid GeoJSON. I'm not sure that this test actually tests the changes in this PR.